src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
```

Comment:
I updated the name of this component to `LocationMenu` in the requirements and #413 because I thought it was more precise, but I think the article I shared about custom vs native select [makes a good case for either](https://css-tricks.com/striking-a-balance-between-native-and-custom-select-elements/#aa-a-note-about-selects-that-look-like-menus).

Reply:
Updated in 3217ecf6f1c65f94030628723e6bed22147f356c.